src/sentry/static/sentry/app/sentryTypes.jsx

```diff
@@ -685,6 +685,7 @@ export const EChartsGrid = PropTypes.shape({
 export const EChartsLegend = PropTypes.shape({
   // Show legend on chart
   show: PropTypes.bool,
+  truncate: PropTypes.oneOfType([PropTypes.bool, PropTypes.number]),
```

Comment:
`truncate` doesn't currently support boolean, though it'd be good if it did apply a default length (same as tooltip)

Reply:
added it 👍